File: Models/Task.cs
namespace taskManager.Models
{
  public class Task
  {
    public string ID { get; set; }
    public string Title { get; set; }
    public string Description { get; set; }
    public bool Important { get; set; }
    public string DueDate { get; set; }
    public string Location { get; set; }
    public string AlertText { get; set; }
    public string Status { get; set; }

    public string User { get; set; }

  }
}

File: wwwroot/js/task.js
const TASK_CONST = {
  ID: '_id',
  TITLE: 'title',
  DESCRIPTION: 'description',
  IMPORTANT: 'important',
  DUE_DATE: 'dueDate',
  LOCATION: 'location',
  ALERT_TEXT: 'alertText',
  STATUS: 'status'
}

class Task {
  constructor(title, description, important, dueDate, location, alertText, status) {
    this.title = title;
    this.description = description;
    this.important = important;
    this.dueDate = dueDate;
    this.location = location;
    this.alertText = alertText;
    this.status = status;
    this.user = "Jerald";
  }
}
